scripts/performance/index-storage/terraform/main.tf

```diff
@@ -0,0 +1,134 @@
+// Copyright 2024 The Sigstore Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+provider "google" {
+    project = var.project
+    zone = var.zone
+    region = var.region
+}
+
+module "network" {
+    source = "git::https://github.com/sigstore/scaffolding.git//terraform/gcp/modules/network?ref=36131f6dd57a0b1d9eaad0f7dd17b11c3ab0eeee"
+
+    region = var.region
+    project_id = var.project
+    cluster_name = "rekor"
+}
+
+module "bastion" {
```

Comment:
Just wondering, is a bastion needed for the tests?

Reply:
The bastion is needed to install the kubernetes services from. It's also easier to run the tests from here instead of setting up a gateway and public URL (and in my case org policies prevented me from doing so).

Reply:
Sounds good, just saw you noted that in the readme.